homeassistant/components/tuya/sensor.py

```diff
@@ -49,28 +49,25 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
 BATTERY_SENSORS: tuple[TuyaSensorEntityDescription, ...] = (
     TuyaSensorEntityDescription(
         key=DPCode.BATTERY_PERCENTAGE,
-        name="Battery",
```

Comment:
Tuya is a special case. For number and sensor entities, we cannot rely on translations from the device class, as (depending on the response of the API), the device class can be removed at runtime. (Resulting in an entity without a translation).